src/index.js

```diff
@@ -0,0 +1,136 @@
+
+module.exports = {
```

Comment:
```suggestion
module.exports = {
  root: true
```
Faut pas ajouter un root: true ?